spec/support/govuk_account_helper.rb

```diff
@@ -0,0 +1,18 @@
+module GovukAccountHelper
+  include SessionPathConcern
+
+  def stub_sign_in_url(**params)
```

Comment:
If the main motivation to have the paths be in a separate concern was so that they could be included in the tests, well, there's only one path now.